src/redux/services/fileEntry.ts

```diff
@@ -503,7 +503,13 @@ export function removePath(absolutePath: string, state: AppState, fileEntry: Fil
     }
   }
 
-  reprocessResources([], state.resourceMap, state.fileMap, state.resourceRefsProcessingOptions, {
-    resourceKinds: removalSideEffect.removedResources.map(r => r.kind),
-  });
+  reprocessResources(
+    Object.values(state.resourceMap).map(r => r.id),
```

Comment:
I'm wondering if we should filter out the preview resources from here since this gets called when a path is removed and that shouldn't affect any of the preview modes.

Reply:
makes sense - i'll improve accordingly

Reply:
fixed..